credentials/utopia-ead-vc-v2/queries.json

```diff
@@ -0,0 +1,23 @@
+{
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your Employment Authorization Document Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/ns/credentials/v2",
+          "https://w3id.org/citizenship/v4rc1",
+          "https://w3id.org/vc/render-method/v2rc1"
```

Comment:
Do we care what version of render method was used when querying for the credential?

```suggestion
```

I wouldn't think so.